scripting/src/main/java/org/infinispan/scripting/impl/ScriptingManagerImpl.java

```diff
@@ -69,7 +69,7 @@ public void initialize(final EmbeddedCacheManager cacheManager, InternalCacheReg
 
    Cache<String, String> getScriptCache() {
       if (scriptCache == null) {
-         scriptCache = cacheManager.getCache(SCRIPT_CACHE);
+         scriptCache = SecurityActions.getUnwrappedCache(cacheManager.getCache(SCRIPT_CACHE));
```

Comment:
Unfortunately I am not familiar with the chain of invocation here, but since the SecurityActions doesn't use a Privileged block does that mean the original invocation of this requires ADMIN privileges?